frontend/packages/console-app/src/components/pdb/AvailabilityRequirement.tsx

```diff
@@ -0,0 +1,28 @@
+import * as React from 'react';
+import * as _ from 'lodash';
+import { useTranslation } from 'react-i18next';
+import { PodDisruptionBudgetKind } from '@console/internal/module/k8s';
+
+const AvailabilityRequirement: React.FC<AvailabilityRequirementProps> = ({ pdb, replicas }) => {
+  const { t } = useTranslation();
+  return (
+    <>
+      {!_.isNil(pdb?.spec?.minAvailable)
+        ? t('console-app~Min available {{minAvailable}} of {{count}} pod', {
+            minAvailable: pdb.spec.minAvailable,
+            count: replicas,
+          })
+        : t('public~Max unavailable {{maxUnavailable}} of {{count}} pod', {
+            maxUnavailable: pdb?.spec?.maxUnavailable ?? 1,
```

Comment:
Can we say this is a safe fallback? "Max unavailable 1 of 1 pod". 

If I have neither min available or max unavailable set, would it not just default to nothing? Ie, `Max unavailable is {replica count} of {replica count} pods`?

I'm not entirely sure what happens -- but as I read the documentation it seems like nothing.

Eg, I think we want to either say nothing or say all are allowed to be unavailable (My Deployment):
![image](https://user-images.githubusercontent.com/8126518/162489705-6cbcf27c-fe8e-46cc-9c3f-074d7f3b6291.png)

Looks odd given my settings.